lib/src/room.dart

```diff
@@ -600,22 +600,29 @@ class Room {
 
   /// Sends a normal text message to this room. Returns the event ID generated
   /// by the server for this message.
-  Future<String?> sendTextEvent(String message,
-      {String? txid,
-      Event? inReplyTo,
-      String? editEventId,
-      bool parseMarkdown = true,
-      bool parseCommands = true,
-      String msgtype = MessageTypes.Text,
-      String? threadRootEventId,
-      String? threadLastEventId}) {
+  Future<String?> sendTextEvent(
+    String message, {
+    String? txid,
+    Event? inReplyTo,
+    String? editEventId,
+    bool parseMarkdown = true,
+    bool parseCommands = true,
+    String msgtype = MessageTypes.Text,
+    String? threadRootEventId,
+    String? threadLastEventId,
+    StringBuffer? commandStdout,
```

Comment:
Can we add a little bit dartdoc comments to make it more intuitive how to use the `StringBuffer`? Or at least a link to the documentation. For example I see this type for the first time and would have to look it up before using it.

Reply:
We might want to have more information from the output as just a String, as errors could also be interesting, while we also might get errors and values at the same time. Could be probably solved by introducing a CommandResult type which contains the success data, (Matrix-)errors as well as the output of the command

Reply:
Hmmmm, in my implementation I simply used the regular error catching around the `Client.parseAndRunCommand()` and `Room.send*` methods for error handling. Since there are decent, unified Error types available I don't see an advantage in a dedicated error collection mechanism ; but if you have any proposal on how to implement some improved error handling, I'm curious for suggestions !

The advantage of a `StringBuffer` is that it basically allows to perform random access (similar to a `File.open` call) and object-oriented append syntax. I simply considered this as way more convenient for this use case - since a String could easily accidentally lose it's reference and doesn't support such a beautiful syntax to access its onwardly growing contents during the asynchronous writing into it.

Reply:
@TheOneWithTheBraid Can we please have dartdoc comment for the `commandStdout` that says it's used for the command output in case a command was executed

Reply:
@TheOneWithTheBraid ping

> @TheOneWithTheBraid Can we please have dartdoc comment for the `commandStdout` that says it's used for the command output in case a command was executed